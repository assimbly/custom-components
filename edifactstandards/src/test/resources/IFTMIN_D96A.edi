UNA:+.? '
UNB+UNOA:2+8714252014808::14+8714252011517::14+130402:1219+24535'
UNH+24546+IFTMIN:D:96A:UN'
BGM+340+347605+9'
DTM+137:20130402:102'
DTM+2:20130404:102'
DTM+200:20130403:102'
TSR+11++N'
FTX+DEL+++EXACT DONDERDAG 4.4 TUSSEN 8.15-12.00 UUR:AFLEVEREN'
TOD+6++CIP'
NAD+DP+++DUMMY B.V.+VERLENGDE VOORBEELDWEG 123+AMSTERDAM++1234AB+670'
NAD+CZ+++DUMMY WIRE & CABLE B.V.+INDUSTRIEGEBIED?: 1234:VOORBEELDWEG 10+AMSTERDAM++1234 AB+670'
CTA+IC+:JANE DOE'
COM+?+31(0)12 3456789:TE'
NAD+SF+++DUMMY WIRE & CABLE B.V.+INDUSTRIEGEBIED?: 1234:VOORBEELDWEG 10+AMSTERDAM++1234 AB+670'
CTA+IC+:JANE DOE'
COM+?+31(0)12 3456789:TE'
NAD+FW+++LOGISTICS B.V.++++1234 AB+670'
GID+1.0+1.0:E827::9'
HAN+UST::9'
MEA+AAE+G+KGM:24.0'
MEA+AAE+GMC+MTQ:80.0'
DIM+1+MTR:41.0:41.0:48.0'
PCI+33E'
GIN+BJ+00387142520086154710'
UNT+98126+24546'
UNZ+1+24535'