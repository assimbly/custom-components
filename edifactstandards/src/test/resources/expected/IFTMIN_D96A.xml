<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<Envelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.blujaysolutions.com/2017/IFTMIN file:///C:/Users/jss/Documents/seacon/germany_interfaces/develop/iftmin_D96A/IFTMIN_D96A.xsd" xmlns="http://www.blujaysolutions.com/2017/IFTMIN">
    <Interchange>
        <UNB>
            <F0020>24535</F0020>
            <S001>
                <F0001>UNOA</F0001>
                <F0002>2</F0002>
            </S001>
            <S002>
                <F0004>8714252014808</F0004>
                <F0008>14</F0008>
            </S002>
            <S003>
                <F0010>8714252011517</F0010>
                <F0014>14</F0014>
            </S003>
            <S004>
                <F0017>130402</F0017>
                <F0019>1219</F0019>
            </S004>
        </UNB>
        <UNZ>
            <F0036>1</F0036>
            <F0020>24535</F0020>
        </UNZ>
        <Group>
            <Message_IFTMIN>
                <UNH>
                    <F0062>24546</F0062>
                    <S009>
                        <F0065>IFTMIN</F0065>
                        <F0052>D</F0052>
                        <F0054>96A</F0054>
                        <F0051>UN</F0051>
                    </S009>
                </UNH>
                <BGM>
                    <F1004>347605</F1004>
                    <F1225>9</F1225>
                    <C002>
                        <F1001>340</F1001>
                    </C002>
                </BGM>
                <DTM>
                    <C507>
                        <F2005>137</F2005>
                        <F2380>20130402</F2380>
                        <F2379>102</F2379>
                    </C507>
                </DTM>
                <DTM>
                    <C507>
                        <F2005>2</F2005>
                        <F2380>20130404</F2380>
                        <F2379>102</F2379>
                    </C507>
                </DTM>
                <DTM>
                    <C507>
                        <F2005>200</F2005>
                        <F2380>20130403</F2380>
                        <F2379>102</F2379>
                    </C507>
                </DTM>
                <TSR>
                    <C536>
                        <F4065>11</F4065>
                    </C536>
                    <C537>
                        <F4219>N</F4219>
                    </C537>
                </TSR>
                <FTX>
                    <F4451>DEL</F4451>
                    <C108>
                        <F4440>EXACT DONDERDAG 4.4 TUSSEN 8.15-12.00 UUR</F4440>
                        <F4440>AFLEVEREN</F4440>
                    </C108>
                </FTX>
                <UNT>
                    <F0074>98126</F0074>
                    <F0062>24546</F0062>
                </UNT>
                <SG2>
                    <TOD>
                        <F4055>6</F4055>
                        <C100>
                            <F4053>CIP</F4053>
                        </C100>
                    </TOD>
                </SG2>
                <SG11>
                    <NAD>
                        <F3035>DP</F3035>
                        <F3164>AMSTERDAM</F3164>
                        <F3251>1234AB</F3251>
                        <F3207>670</F3207>
                        <C080>
                            <F3036>DUMMY B.V.</F3036>
                        </C080>
                        <C059>
                            <F3042>VERLENGDE VOORBEELDWEG 123</F3042>
                        </C059>
                    </NAD>
                </SG11>
                <SG11>
                    <NAD>
                        <F3035>CZ</F3035>
                        <F3164>AMSTERDAM</F3164>
                        <F3251>1234 AB</F3251>
                        <F3207>670</F3207>
                        <C080>
                            <F3036>DUMMY WIRE &amp; CABLE B.V.</F3036>
                        </C080>
                        <C059>
                            <F3042>INDUSTRIEGEBIED: 1234</F3042>
                            <F3042>VOORBEELDWEG 10</F3042>
                        </C059>
                    </NAD>
                    <SG12>
                        <CTA>
                            <F3139>IC</F3139>
                            <C056>
                                <F3412>JANE DOE</F3412>
                            </C056>
                        </CTA>
                        <COM>
                            <C076>
                                <F3148>+31(0)12 3456789</F3148>
                                <F3155>TE</F3155>
                            </C076>
                        </COM>
                    </SG12>
                </SG11>
                <SG11>
                    <NAD>
                        <F3035>SF</F3035>
                        <F3164>AMSTERDAM</F3164>
                        <F3251>1234 AB</F3251>
                        <F3207>670</F3207>
                        <C080>
                            <F3036>DUMMY WIRE &amp; CABLE B.V.</F3036>
                        </C080>
                        <C059>
                            <F3042>INDUSTRIEGEBIED: 1234</F3042>
                            <F3042>VOORBEELDWEG 10</F3042>
                        </C059>
                    </NAD>
                    <SG12>
                        <CTA>
                            <F3139>IC</F3139>
                            <C056>
                                <F3412>JANE DOE</F3412>
                            </C056>
                        </CTA>
                        <COM>
                            <C076>
                                <F3148>+31(0)12 3456789</F3148>
                                <F3155>TE</F3155>
                            </C076>
                        </COM>
                    </SG12>
                </SG11>
                <SG11>
                    <NAD>
                        <F3035>FW</F3035>
                        <F3251>1234 AB</F3251>
                        <F3207>670</F3207>
                        <C080>
                            <F3036>LOGISTICS B.V.</F3036>
                        </C080>
                    </NAD>
                </SG11>
                <SG18>
                    <GID>
                        <F1496>1</F1496>
                        <C213>
                            <F7224>1</F7224>
                            <F7065>E827</F7065>
                            <F3055>9</F3055>
                        </C213>
                    </GID>
                    <HAN>
                        <C524>
                            <F4079>UST</F4079>
                            <F3055>9</F3055>
                        </C524>
                    </HAN>
                    <SG20>
                        <MEA>
                            <F6311>AAE</F6311>
                            <C502>
                                <F6313>G</F6313>
                            </C502>
                            <C174>
                                <F6411>KGM</F6411>
                                <F6314>24</F6314>
                            </C174>
                        </MEA>
                    </SG20>
                    <SG20>
                        <MEA>
                            <F6311>AAE</F6311>
                            <C502>
                                <F6313>GMC</F6313>
                            </C502>
                            <C174>
                                <F6411>MTQ</F6411>
                                <F6314>80</F6314>
                            </C174>
                        </MEA>
                    </SG20>
                    <SG21>
                        <DIM>
                            <F6145>1</F6145>
                            <C211>
                                <F6411>MTR</F6411>
                                <F6168>41</F6168>
                                <F6140>41</F6140>
                                <F6008>48</F6008>
                            </C211>
                        </DIM>
                    </SG21>
                    <SG23>
                        <PCI>
                            <F4233>33E</F4233>
                        </PCI>
                        <GIN>
                            <F7405>BJ</F7405>
                            <C208>
                                <F7402>00387142520086154710</F7402>
                            </C208>
                        </GIN>
                    </SG23>
                </SG18>
            </Message_IFTMIN>
        </Group>
    </Interchange>
</Envelope>